manticore/core/cpu/abstractcpu.py

```diff
@@ -789,6 +790,11 @@ def emulate(self, insn):
         emu = UnicornEmulator(self)
         try:
             emu.emulate(insn)
+        except unicorn.UcError as e:
+            text_bytes = ' '.join('%02x'%x for x in insn.bytes)
```

Comment:
we should check here to make sure that it actually is an invalid insn error, instead of another unicorn error. we should be able to do this by checking that the `e.errno` value is `unicorn.UC_ERR_INSN_INVALID`!